exercises/concept/regular-chatbot/.meta/design.md

```diff
@@ -0,0 +1,33 @@
+# Design
+
+## Learning objectives
+
+- Learn about the flavour of JavaScript regarding Regular Expressions.
+- How to create a regular expression, with regular expression literal and RexExp
+- How to pass flags in both versions
+- How to use the most common regex related functions, with special focus on test and match.
+- Understanding the different parts of the match result, incl. capture results
+- Being aware of performance implications
+
+## Out of scope
+
+This exercise is intended for an Introduction to the flavour of Regular Expressions in JavaScript. The main focus resides in understanding the syntax behind the creation and manipulation of regex.
+
+Explaining how to write regular expressions themselves is out of scope for the concept here but we should link to some good resource a student could read to learn about them from scratch. We don't do this as part of the concept because Exercism assumes the student is already fluent in another language and most languages include some form of regular expressions.
+
+## Concepts
```

Comment:
Below should only be a list of concepts the exercise teaches, so only one bullet point with `regular-expressions` in this case. The details of the concept are in the learning objectives above (you can edit those if needed).